source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1073,6 +1081,10 @@ AdaptiveNamespaceStart
         // Buttons go into body panel, show cards go into outer panel so they're not inside the padding
         XamlHelpers::AppendXamlElementToPanel(actionsPanel.Get(), bodyPanel);
         XamlHelpers::AppendXamlElementToPanel(showCardsStackPanel.Get(), parentPanel);
+        // XamlHelpers::AddRow(actionsPanel.Get(), parentGrid, { 1, GridUnitType::GridUnitType_Auto });
+
+        // TODO: EdgeToEdge show cards should not go in "parentPanel", which has margins applied to it from the adaptive card options
```

Comment:
Remove this old code.

Reply:
You can remove this TODO as well, this has been handled in a previous change.

---
In reply to: [182560224](https://github.com/Microsoft/AdaptiveCards/pull/1377#discussion_r182560224) [](ancestors = 182560224)